src/IdeaStatiCa.Plugin/BIMPluginHosting.cs

```diff
@@ -156,8 +194,24 @@ private void SelfServiceHost_Opened(object sender, EventArgs e)
 			}
 		}
 
+		private void SelfServiceHost_UnknownMessageReceived(object sender, UnknownMessageReceivedEventArgs e)
+		{
+			ideaLogger.LogDebug($"SelfServiceHost_UnknownMessageReceived service '{ServiceBaseAddress}'");
```

Comment:
it is a wcf handler.
it is never called in case of our BIM links. It is probably called when the service gets a request which is unknown - wrong data are sent to service.